crates/nu-command/tests/commands/select.rs

```diff
@@ -166,10 +166,9 @@ fn select_ignores_errors_succesfully1() {
         cwd: ".", pipeline(
         r#"
         [{a: 1, b: 2} {a: 3, b: 5} {a: 3}] | select -i b
-            "#
+        "#
     ));
 
-    assert!(actual.out.is_empty());
```

Comment:
Can you make the test check at least something? You could add `| length` at the end of the test and check for `3` or something like that at least.

Reply:
> Can you make the test check at least something? You could add `| length` at the end of the test and check for `3` or something like that at least.

You are right, now I have make changes